pages/clustering/high-availability.mdx

```diff
@@ -463,70 +463,91 @@ You can directly use initialization file `HA_init.cypherl`:
 
 ```
 
-ADD COORDINATOR 2 WITH CONFIG {"bolt_server": "localhost:7691", "coordinator_server": "localhost:10112"};
-ADD COORDINATOR 3 WITH CONFIG {"bolt_server": "localhost:7692", "coordinator_server": "localhost:10113"};
+ADD COORDINATOR 2 WITH CONFIG {"bolt_server": "coord2:7691", "coordinator_server": "coord2:10112"};
+ADD COORDINATOR 3 WITH CONFIG {"bolt_server": "coord3:7692", "coordinator_server": "coord3:10113"};
 
-REGISTER INSTANCE instance_1 WITH CONFIG {"bolt_server": "localhost:7687", "management_server": "localhost:10011", "replication_server": "localhost:10001"};
-REGISTER INSTANCE instance_2 WITH CONFIG {"bolt_server": "localhost:7688", "management_server": "localhost:10012", "replication_server": "localhost:10002"};
-REGISTER INSTANCE instance_3 WITH CONFIG {"bolt_server": "localhost:7689", "management_server": "localhost:10013", "replication_server": "localhost:10003"};
+REGISTER INSTANCE instance_1 WITH CONFIG {"bolt_server": "instance1:7687", "management_server": "instance1:10011", "replication_server": "instance1:10001"};
+REGISTER INSTANCE instance_2 WITH CONFIG {"bolt_server": "instance2:7688", "management_server": "instance2:10012", "replication_server": "instance2:10002"};
+REGISTER INSTANCE instance_3 WITH CONFIG {"bolt_server": "instance3:7689", "management_server": "instance3:10013", "replication_server": "instance3:10003"};
 SET INSTANCE instance_3 TO MAIN;
 
 ```
 
 You can directly use the following `docker-compose.yml` to start the cluster using `docker compose up`:
 
 ```
+networks:
+  memgraph_ha:
+    name: memgraph_ha
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: "172.21.0.0/16"
+
 services:
   coord1:
-    image: "memgraph/memgraph-mage"
+    image: "memgraph/memgraph"
     container_name: coord1
     volumes:
-      - ./license.cypherl:/tmp/init/license.cypherl:ro
-      - ./HA_register.cypherl:/tmp/init/HA_init.cypherl:ro
+      - ./license.cypher:/tmp/init/license.cypher:ro
+      - ./HA_register.cypher:/tmp/init/HA_register.cypher:ro
```

Comment:
Any specific reason why it's .cypher and not .cypherl?

Reply:
it's the same

Reply:
Also, using bind mounts is not a good practice with Docker anymore. It would be better to use Dockerfile and c/p cypherl into the container before staring it.

Reply:
@as51340 but you are using cypherl above as an example and cypher below. I think only one should be used.

Reply:
solved on call